lib/glimesh_web/templates/oauth2_provider/authorization/error.html.eex

```diff
@@ -0,0 +1,7 @@
+<div class="container" style="text-align:center;">
+  <h1>An error has occurred</h1>
```

Comment:
Recommend gettexting this with our Oops error that way translators don't get a new string :)